### DISCUSSION TITLE
Files with extension .indt and .indd not showing in search resualts

### DISCUSSION THREAD

**Discussion by martinedvard** [microsoft-search/pnp-modern-search#4102] [General]
Hi!

I have a simple search on a sharpoint webpage where i search in a specific folder on the intranet. 
It shows all files with doc,dotx and so on but NOT the files with extension .indd and .indt.

Why?

Thanks
Martin

**Comment by martinedvard** [comment]
No one knows :(

**Reply by kasperbolarsen** [reply]
No, I haven't had any issues where a specific filetype is not picked up by the indexer. Well, I once had a similar issue, but in that case the files were password protected, yet the customer expected that the indexer somehow could access the content anyway......

**Reply by martinedvard** [reply]
I think it have something to do with the Result source ID. I use LocalSharePointResults. But if i change to RecentlyChangedItems it shows my files but is dont show just my search resulats but all files. Hmm.. dont know what to do. Im not very good at sharpoint and programing.

**Reply by kasperbolarsen** [reply]
I would recommend staying with LocalSharePointResults and sort by last modified if that is what you are looking for

**Reply by martinedvard** [reply]
I want all my files to show. Now all files show but not .indd and .indt.     Seems to be a bug in the PnP search

**Reply by kasperbolarsen** [reply]
That would be rushing to a conclusion, if you haven't checked what the API returns. I suggest looking at data with one of these debug tools in order to ensure that the API returns what you expect: https://microsoft-search.github.io/pnp-modern-search/using-query-tools-to-verify-issues/

**Reply by wobba** [reply]
@martinedvard You need to verify the KQL you are using to ensure everything comes back. And also verify that the files are indexed and will return in e.g. library search. It's YOU which define the search query, not the web parts. A simple test query would be `FileType:indd`.

**Comment by martinedvard** [comment]
I use filetype:indd nothing shows. All other fileformat shows perfekt. Its just adobe fileformat. I tryed the DEBUG tool but dont understand anything it returns. Just code. Nothing usefull for me. But under the primary result it fins the .indt file but it dont show the other 200 files i have. Im not very good at this. It seams to hard to get this to work så maby we cant use it :(

**Reply by wobba** [reply]
If nothing is returned using `filetype:indd` then it might not be indexed. Checking the crawl log for the file is the next step or you need to file a support ticket - assuming the file is not returned if you search in the Library itself as well.

Using https://pnp.github.io/powershell/cmdlets/Get-PnPSearchCrawlLog.html for example. For filter you should do including the library, not necessarily the full file name due to how different properties are stored in search.

`-Filter "https://contoso.sharepoint.com/sites/site/shared documents` as an example.

And the debug mode is not code, it's data in JSON format ;)
